Makefile

```diff
@@ -160,10 +171,57 @@ run-static-analisys:
 run-unit-test:
 	echo TODO: $(@)
 
+monitor-r53-connection:
+	sleep 10
+	$(APPLICATION_TEST_DIR)/monitor/monitor.sh $(POSTCODE_ENDPOINT)
```

Comment:
Can we move the code in the monitor.sh file into the make target please?  This will make it easier for us to port these same changes across into the Fuzzy API later.